stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../assets/scss/mixins';
+@import '../../assets/scss/breakpoints';
+@import '../../assets/scss/variables';
+
+#toggleSearch {
+  background: none;
+  border: 0;
+  border-bottom: 1px solid $color-black;
+  cursor: pointer;
+  padding-bottom: 10px;
+  text-decoration: underline;
+}
+
+#select-wrapper {
+  display: none;
+
+  &.show-filter {
+    display: block;
+  }
+  .multi-select { padding-top: 24px; }
+}
+
+#selectedchips {
+  padding-top: 24px;
+
```

Comment:
@Ashish-srij Please don't use much id's for the styling as we should use classes for styling and id's for JS

Reply:
Done